R/utils.R

```diff
@@ -123,10 +123,39 @@ same_contents <- function(path, contents) {
   identical(text_hash, file_hash)
 }
 
-r_files <- function(path) {
+all_r_files <- function(path) {
   sort_c(dir(file.path(path, "R"), "[.Rr]$", full.names = TRUE))
 }
 
+r_files <- function(path) {
+    rbuildignore <- file.path(path, ".Rbuildignore")
+    rfiles <- all_r_files(path)
+    if (file.exists(rbuildignore)) {
```

Comment:
This should to be pulled out into a separate function, called something like `ignore_files`